cassandra/datadog_checks/cassandra/data/metrics.yaml

```diff
@@ -216,43 +216,43 @@ jmx_metrics:
       attribute:
         CollectionCount:
           metric_type: counter
-          alias: jmx.gc.minor_collection_count
+          alias: jvm.gc.minor_collection_count
```

Comment:
Do we need those metrics at all ? They are already present here: https://github.com/DataDog/jmxfetch/blob/7e381445770ad60226397385fb2e7d4df9b3a739/src/main/resources/org/datadog/jmxfetch/new-gc-default-jmx-metrics.yaml

https://github.com/DataDog/jmxfetch/blob/c25dcc32b2c36c591e8acad668d121ff2c5a05eb/src/main/java/org/datadog/jmxfetch/Instance.java#L223-L226

and can be enabled like this:

```
init_config:
  is_jmx: true
  new_gc_metrics: true
```

Reply:
Should we remove them here and enable `new_gc_metrics` by default?